kable-core/src/androidMain/kotlin/gatt/Callback.kt

```diff
@@ -275,6 +273,11 @@ internal class Callback(
         if (status == GATT_SUCCESS) this.mtu.value = mtu
     }
 
+    override fun onServiceChanged(gatt: BluetoothGatt) {
+        logger.debug { message = "onServiceChanged" }
+        // todo
```

Comment:
Intentionally vague?

Reply:
This is just a stub for future work.
But good call-out, I'll update it to link https://github.com/JuulLabs/kable/issues/662.